extension/js/page-navigator.js

```diff
@@ -25,9 +25,12 @@
 // (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 // SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-function PageNavigator(base_image_uri) {
+function PageNavigator(base_image_uri, settings) {
```

Comment:
I've added settings as an optional variable here, used only in calls from thread pages. Can just as easily add it to the other calls, but if a reference isn't needed, I didn't see the point in adding one.

I passed settings instead of just the individual one we need because I'd like to eventually add a feature from SALR Firefox where we could have another setting to add a bookmark star/status in the page navigator.